src/managers/ProjectManager.ts

```diff
@@ -601,7 +601,13 @@ export class ComponentLibraryProject extends Project {
             ],
             from: /uri\s*=\s*"(.+)\.brs"/gi,
             to: (match) => {
-                return match.replace('.brs', this.postfix + '.brs');
+
+                // do not alter file ending if it's an external library eg with a common:/ file path
+                if (match.toLowerCase().startsWith('uri="common:/')) {
```

Comment:
Thinking out loud here but I wonder if it should be any file protocol other than pkg? 

This makes the logic a bit more tricky because you wouldn't want it to match on relative paths but updating it to would this way would mean we also could cover other unknown types. Let hypothetically say Roku has some "secret:/" file protocol name for example.

Reply:
I agree with you chris. 

And I think the logic should be fairly straightforward, we just need a regex that will match any non pkg and non relative path.